Tests/Tests/STPPaymentIntentFunctionalTest.swift

```diff
@@ -128,5 +128,78 @@ class STPPaymentIntentFunctionalTestSwift: XCTestCase {
             XCTAssertNotNil(clientSecret)
         }
     }
-    
+
+    /// Setting SFU to `true` when a customer is set should set the parameter to `off_session`.
+    func testPaymentIntentParamsWithSFUTrueAndCustomer() {
```

Comment:
I'm surprised to see this test in here b/c this logic is very specific to PaymentSheet, maybe we can move it somewhere like `PaymentSheet+APITest.swift`

Reply:
I had put them here because these tests are related to PaymentIntent, but your suggestion makes sense, I moved them.